Assets/Scripts/Combat/Gun.cs

```diff
@@ -18,9 +18,9 @@ public class Gun : MonoBehaviour
     private Transform firePoint;
 
     [SerializeField]
-    private ParticleSystem muzzleFlash;
+    private ParticleSystem muzzleParticle;
 
-    // Update is called once per frame
+    // checks to see if lmb is clicked to fire the weapon and play audiosource attached
     void Update()
     {
         timer += Time.deltaTime;
```

Comment:
Maybe create a serialized field for timerDuration so that you can change it in the editor and try testing different amounts of time